Sources/StreamChatUI/Gallery/VideoPlaybackControlView.swift

```diff
@@ -52,6 +52,7 @@ open class VideoPlaybackControlView: _View, ThemeProvider {
     private var playerStatusObserver: NSKeyValueObservation?
     private var playerItemObserver: NSKeyValueObservation?
     private var itemDurationObserver: NSKeyValueObservation?
+    private var _currentPlayer: AVPlayer? { player }
```

Comment:
I would move this closer to the deinit method, and provide a comment to explain this is to overcome a Swift 6 warning 👍

Reply:
Done 👍